webextensions/api/action.json

```
@@ -466,7 +466,7 @@
             "support": {
               "chrome": {
                 "version_added": "118",
-                "notes": "Is only available to policy installed extensions and dev builds (e.g., Canary)."
+                "notes": "Is only available to policy installed extensions and dev builds (e.g., Canary) between Chrome 118 and Chrome 126."
```

I think we should do two things here:

1. Shorten the note as follows.
2. Ideally split this up into two support statements:
    - A partial_implementation 118-127 with this note.
    - A full implementation from 127.

```suggestion
                "notes": "Before Chrome 127, it is only available to policy installed extensions."
```

---

i've update it to split the desc into 2 parts, as I think it will be a little bit more clear? :)